crates/lib3h/src/gateway/gateway_transport.rs

```diff
@@ -18,6 +18,14 @@ use serde::{Deserialize, Serialize};
 impl P2pGateway {
     /// Handle IncomingConnection event from child transport
     fn handle_incoming_connection(&mut self, span: Span, uri: Lib3hUri) -> TransportResult<()> {
+        // TODO: This is prbably wrong in that a different level of URI should be being bubbled up.
```

Comment:
Is there follow up work documented for this?

Reply:
done